### DISCUSSION TITLE
Problem with knit

### DISCUSSION THREAD

**Discussion by danielsgranja** [munoztd0/DAP_2021#25] [General]
Hello !

I have a problem to knit my rmd. It was working yesterday and now it tells me that:

Quitting from lines 179-180 (DAP_v4-1-22.Rmd) 
Erreur dans get(name, envir = knit_global(), mode = mode, inherits = FALSE) : 
  objet '.DAP_v4-1-22_cache/html/random slope_3459bedde1b8fc873192420dcd3e66a7' de mode 'character' introuvable
Appels : <Anonymous> ... process_group.block -> call_block -> <Anonymous> -> get
Exécution arrêtée

It's weird because the line is simply :
anova(m2.2, m3.2)

So I don't know what to do.

Sorry to bother you so late ^^'

edit: everything works when i run the code, it's only the knit that do not work

**Comment by munoztd0** [comment]
I can't check but try to remove the file in the cache folder and rerun

David

**Reply by danielsgranja** [reply]
It worked, thank you!
